dll/win32/uxtheme/CMakeLists.txt

```diff
@@ -8,7 +8,14 @@ list(APPEND SOURCE
     draw.c
     main.c
     metric.c
-    msstyles.c
+    msstyles.c)
+
+if(DLL_EXPORT_VERSION GREATER_EQUAL 0x600)
+    list(APPEND SOURCE
+        pngsup.cpp)
+endif()
+
+list(APPEND SOURCE
```

Comment:
You now have 3 lists here, just add the new one after the first unconditional one.